### ISSUE TITLE
Missing jquery.mobile.custom.js in 2.4.4

### ISSUE THREAD

**Issue by mohitnavik** [magento/magento2#35864]
### Preconditions and environment

I have upgraded Magento from 2.4.3-p2 to 2.4.4. After upgrading the magento version, I am getting "Uncaught Error: Script error for jquery/jquery.mobile.custom" error in the console.

![image](https://user-images.githubusercontent.com/41897612/182595682-471ba388-ea25-4154-a9b2-99850569c071.png)

I have custom theme and added the following in requirejs-config.js

_var config = {
    map: {
        '*': {
            'rowBuilder':             'Magento_Theme/js/row-builder',
            'toggleAdvanced':         'mage/toggle'
        }
    },
    paths: {
        'jquery/ui': 'jquery/jquery-ui',
        'countdown': "js/countdown.min"
    },
    shim: {
        'countdown': {
            'deps': ['jquery']
        }
    },
    deps: [
        'jquery/jquery.mobile.custom',
        'mage/common',
        'mage/dataPost',
        'mage/bootstrap'
    ],
    config: {
        mixins: {
            'Magento_Theme/js/view/breadcrumbs': {
                'Magento_Theme/js/view/add-home-breadcrumb': true
            },
            'Magento_Ui/js/modal/modal': {
                'Magento_Ui/js/model/modal-mixin': true
            }
        }
    }
};_

I have checked and found that "lib/web/jquery/jquery.mobile.custom.js" is missing.

### Steps to reproduce

1. Upgrade from 2.4.3-p2 to 2.4.4
2. Add custom code in requirejs-config.js as per shared.
3. Go to frontend(any page) and check browser console.

### Expected result

Error should not be show in console or alternate way to fix it

### Actual result

Error in console: Uncaught Error: Script error for jquery/jquery.mobile.custom

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by mohitnavik** [comment]
Hi @engcom-Lima,

Thanks for your response

Is **"jquery.mobile.custom.js"**  file in the 2.4.4?

Did you try with new custom theme?

Thanks,

**Comment by engcom-Lima** [comment]
Hi @mohitnavik ,

Yes I have created custom theme as you can see my screeshot in above [comment](https://github.com/magento/magento2/issues/35864#issuecomment-1205318184). I have checked there is no `jquery.mobile.custom.js` file in latest `2.4-develop` branch. 
Can you please test it in the latest `2.4-develop` if you are facing any issue let us know.

Thanks

**Comment by engcom-Lima** [comment]
Hi @mohitnavik ,

We have noticed that this issue has not been updated for a period of 14 Days.
Hence we assume that this issue is fixed now, so we are closing it.Please raise a fresh ticket or reopen this ticket if you need more assistance on this.

Thanks

**Comment by Vitaliy-Lazarev** [comment]
Hi everyone. 2.4.5-p1 has the same issue. The jquery.mobile.custom.js file doesn't exist.

**Comment by technodelight** [comment]
This is still reproducible on 2.4.4-p2 as well

**Comment by ciancastillo** [comment]
Facing the same issue in 2.4.5-p1

**Comment by ponaravind-prevaj** [comment]
Im also facing the same issue in magento open source 2.4.5

**Comment by vahir2016** [comment]
I have the same issue but I had a guess.

The file had been removed from magento 2.4.4 (check this https://magento.stackexchange.com/questions/355000/jquery-mobile-custom-js-file-error-after-upgrade-to-magento-2-4-4).

But if you are upgrading from an old magento the file is still located in lib/web/jquery/jquery.mobile.custom.js so it will works until you will remove the file.

In my case I had to remove the file and also remove the reference from src/app/design/frontend/XXX/YYY/web/mage/menu.js to the file in the define.
